gremlin-driver/src/test/java/org/apache/tinkerpop/gremlin/util/PerformanceTest.java

```diff
@@ -53,18 +55,18 @@ class PerformanceTest {
     private static final List<Integer> POOL_SIZE = new ArrayList<>();
     private static final List<Integer> POOL_QUERY_COUNT = new ArrayList<>();
     static {
-        // POOL_SIZE.add(1);
+        POOL_SIZE.add(1);
         POOL_SIZE.add(2);
         POOL_SIZE.add(4);
         POOL_SIZE.add(8);
-        // POOL_QUERY_COUNT.add(50);
-        // POOL_QUERY_COUNT.add(100);
+        POOL_QUERY_COUNT.add(50);
+        POOL_QUERY_COUNT.add(100);
         POOL_QUERY_COUNT.add(250);
-        // POOL_QUERY_COUNT.add(500);
+        POOL_QUERY_COUNT.add(500);
     }
 
-    public static void main(String[] args) {
-        executePerformanceTests();
+    private static void main(String[] args) {
+        // executePerformanceTests();
```

Comment:
Why is this commented out?

Reply:
There is no circumstance where it should be run outside of some manual tests, and will be removed in the near future. It's just here for reference right now.